decidim-core/lib/decidim/acts_as_author.rb

```diff
@@ -14,7 +14,7 @@ module ActsAsAuthor
       #
       # Return: The presenter for the current author.
       def presenter
-        raise NotImlementedError, "Authors must return an instance of its Presenter via this method."
+        raise NotImplementedError, "Authors must return an instance of its Presenter via this method."
```

Comment:
This is notable. It isn't the first time I've run into a test (in Ruby!) like this.

Reply:
good catch!